Formula/g/g-ls.rb

```diff
@@ -0,0 +1,23 @@
+class GLs < Formula
+  desc "Powerful and cross-platform ls"
+  homepage "https://g.equationzhao.space"
+  url "https://github.com/Equationzhao/g/archive/refs/tags/v0.25.3.tar.gz"
+  sha256 "4c4fb3a1790cf4635795925d93f69db624a56f6d9abd8eb9a634c18a7be73f7a"
+  license "MIT"
+  head "https://github.com/Equationzhao/g.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(output: "g", ldflags: "-s -w")
+    bin.install "g"
```

Comment:
Small nit, otherwise lgtm:
```suggestion
    system "go", "build", *std_go_args(output: bin/"g", ldflags: "-s -w")
```

Reply:
good call 👍